docs/docs/statistics/powerTechnical.mdx

```diff
@@ -0,0 +1,346 @@
+---
+title: Power Analysis Technical Details
+description: Technical details of power analysis
+sidebar_label: Power Technical
+slug: powerTechnical
```

Comment:
Follow the naming convention of all of the other docs. This should be `power-technical.mdx` and `power-technical` should be the slug.

Reply:
done